# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../twoftpd/compile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- twoftpd-1.20/Makefile.vanilla	2006-10-04 15:56:44.000000000 +0000
+++ twoftpd-1.20/Makefile	2006-10-04 15:57:31.000000000 +0000
@@ -146,7 +146,7 @@
 	./compile twoftpd-anon.c
 
 twoftpd-auth: load twoftpd-auth.o main.a socket.lib
-	./load twoftpd-auth main.a -lcvm-client -lbg-iobuf -lbg-net -lbg-unix `cat socket.lib`
+	./load twoftpd-auth main.a -lcvm-client -lbg-str -lbg-iobuf -lbg-net -lbg-unix `cat socket.lib`
 
 twoftpd-auth.o: compile twoftpd-auth.c twoftpd.h
 	./compile twoftpd-auth.c
